prime-router/src/main/kotlin/fhirengine/translation/hl7/config/ContextConfig.kt

```diff
@@ -0,0 +1,19 @@
+package gov.cdc.prime.router.fhirengine.translation.hl7.config
+
+import gov.cdc.prime.router.Hl7Configuration
+import gov.cdc.prime.router.Receiver
+
+/**
+ * Top level configuration interface passed around with CustomContext
+ */
+sealed interface ContextConfig
+
+/**
+ * HL7 specific custom context configuration
+ */
+data class HL7TranslationConfig(
+    val hl7Configuration: Hl7Configuration,
+    val receiver: Receiver?
+) : ContextConfig {
```

Comment:
I don't know why I didn't catch this earlier, I didn't realize this class was being used inside the FHIR -> HL7 converter library. We need to  move `HL7TranslationConfig` outside the translation/hl7 folder

Reply:
Where should these types of files live? still within the `fhirengine` package or outside that entirely?